build/build-changelog.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$UPDATE = " - Added support for latest API updates.";
+$path = __DIR__ . '/../CHANGELOG.md';
+
+$diff = shell_exec("git status --porcelain");
```

Comment:
Thanks for notice! It actually works at my local v2.8.1, and its [original commit](https://github.com/git/git/commit/6f15787181a163e158c6fee1d79085b97692ac2f) merged in 2009, I'll find our the initial version that this feature comes out.